packages/google_maps_flutter/google_maps_flutter_web/lib/src/dom_window_extension.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js_interop';
+
+import 'package:web/web.dart' as web;
+
+/// This extension gives [web.Window] a nullable getter to the `trustedTypes`
+/// property, which is used to check for feature support.
+extension NullableTrustedTypesGetter on web.Window {
+  /// (Nullable) Bindings to window.trustedTypes.
+  ///
+  /// This may be null if the browser doesn't support the Trusted Types API.
+  ///
+  /// See: https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API
+  @JS('trustedTypes')
+  external GoogleMapsTrustedTypePolicyFactory? get nullableTrustedTypes;
+}
+
+// TODO(ditman): remove this extension type when we depend on package:web 0.5.1
+/// This extension exists as a stop gap until `package:web 0.5.1` is released.
+/// That version provides the `TrustedTypes` API.
+@JS('TrustedTypePolicyFactory')
```

Comment:
These types are prefixed to avoid name collisions with `package:web` types.

Reply:
Thanks!